dune

```diff
@@ -1,4 +0,0 @@
-(env
- (_
-  (binaries
-   (test/expect/expect_test.exe as expect-test))))
```

Comment:
Why is it better / necessary not to make the executable available as "expect-test"?

Reply:
I was getting error from dunes on older compiler since the stanza responsible for building `expect_test.exe` was disabled.

It might be the case that the error was only triggered because some rules were requiring `expect-test`, I'll give it a spin just to make sure.

Reply:
Well it turns out it wasn't required and was indeed only triggered if an active rule depended on `expect-test` so I reverted this part!

Good catch!